library/core/src/ffi/mod.rs

```diff
@@ -49,163 +49,18 @@ macro_rules! type_alias {
     }
 }
```

Comment:
Forgot to mention, the macro `type_alias` needs to move to `primitives.rs` as well. Otherwise the rmake test will fail.